cluster/common.sh

```diff
@@ -386,703 +306,6 @@ function find-release-tars() {
   fi
 }
 
-# Discover the git version of the current build package
-#
-# Assumed vars:
-#   KUBE_ROOT
-# Vars set:
-#   KUBE_GIT_VERSION
-function find-release-version() {
```

Comment:
I didn't see any uses of this function so I just deleted it.

Reply:
Looks like @mikedanese removed the last file using it in 4708714dd10b17f02693ae36ff6e3bd073d8e064 (a year ago). So probably fine to delete.